presto-hive/src/test/java/com/facebook/presto/hive/TestShowStats.java

```diff
@@ -106,6 +106,22 @@ public void testShowStats()
                         "   (null, null, null, null, 5.0, null, null))");
     }
 
+    @Test
+    public void testShowStatsNotSelecAllColumns()
+    {
+        assertQueryFails("SHOW STATS FOR (SELECT nationkey FROM nation_partitioned)", ".*Only SELECT \\* is supported in SHOW STATS SELECT clause");
+    }
+
+    @Test
+    public void testShowStatsWhereClause()
```

Comment:
can you add a test query that only uses the partitioning key, but the filter still can't be pushed down because it contains some complex expression or an `or` expression?